src/timelines/taskTrial.js

```diff
@@ -1,33 +1,44 @@
 // import trials
-import fixation from '../trials/fixation'
-import showCondition from '../trials/showCondition'
-import taskEnd from '../trials/taskEnd'
-
+import { showMessage, fixation } from "@brown-ccv/behavioral-task-trials";
+import { config } from "../config/main";
+import { eventCodes } from "../config/main";
+import { earningsDisplay } from "../lib/markup/earnings";
 
 const taskTrial = (blockSettings, blockDetails, condition) => {
-  // initialize trial details
-  let trialDetails = {
-    condition: condition,
-    trial_earnings: 0,
-    start_time: Date.now()
-  }
-
   // timeline
   let timeline = [
     // fixation
-    fixation(650),
+    fixation(config, {
+      responseType: "html_keyboard_response",
```

Comment:
does fixation need a passed response type? I'd think the default is almost always fine